docs/docs/production.rst

```diff
@@ -0,0 +1,58 @@
+.. _production:
+
+.. note::
+
+   To use this tutorial, you need to :ref:`install unitxt <install_unitxt>`.
+
+=====================================
+Using in production
+=====================================
+
+Unitxt can be used to process data in production. First define a recipe:
+
+.. code-block:: python
+
+  recipe = "card=cards.wnli,template=templates.classification.multi_class.relation.default,demos_pool_size=5,num_demos=2"
+
+
+Second prepare an instance in the exact schema of the task in that recipe:
+
+
+.. code-block:: python
+
+  instance = {
```

Comment:
is there a way from a recipe to know its from a speficic task?
I want the metric to only accept cards that are from specific task, so I know the recipe beforehand?